test/100-basic-name-resolution.bats

```diff
@@ -165,3 +165,26 @@ load helpers
 	# contain unexpected warning.
 	assert "$output" !~ "WARNING: recursion requested but not available"
 }
+
+# Internal network, meaning no DNS servers.
+# Hence all external requests must fail.
+@test "basic container - internal network has no DNS" {
+	setup_slirp4netns
+
+	subnet_a=$(random_subnet 5)
```

Comment:
I See that all the other cases have this as well but it is wrong, for a ipv4 subnet do not call it with an argument, the only valid argument for this function is 6 for ipv6.